SpotifyAPI/Web/Auth/WebApiFactory.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using SpotifyAPI.Web.Enums;
+using SpotifyAPI.Web.Models;
+
+namespace SpotifyAPI.Web.Auth
+{
+    public class WebApiFactory
```

Comment:
Just commited the code style changes as requested. (Most were due to my ReSharper-settings... ;-) )